src/firmware_new/proto/BUILD

```diff
@@ -1,30 +1,17 @@
-load("@rules_proto//proto:defs.bzl", "proto_lang_toolchain", "proto_library")
-load("@com_google_protobuf//:protobuf.bzl", "proto_gen")
-load(":nanopb.bzl", "c_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+load("//firmware_new:nanopb.bzl", "nanopb_proto_library")
 
 package(default_visibility = ["//visibility:public"])
 
-# add more proto here
-proto_files = [
-    "control.proto",
-]
-
-# proto files
 proto_library(
     name = "fw_proto",
-    srcs = proto_files,
+    srcs = ["control.proto"],
```

Comment:
could you try building `tbots_robot_msg.proto` and `components.proto`? Thats the big one because of the `#include` statements, couldn't get that to work easily with the macro

Reply:
Sorry, should have mentioned this above, but I did confirm that `#include`'s work with this rule.